utils/install_tpm_libs_rhel.sh

```diff
@@ -1,7 +1,7 @@
-TPM2_TSS_VER="3.0.3"
+TPM2_TSS_VER="4.0.0"
 TPM2_TSS_LINK="https://github.com/tpm2-software/tpm2-tss/releases/download/$TPM2_TSS_VER/tpm2-tss-$TPM2_TSS_VER.tar.gz"
-TPM2_TSS_ENGINE_VER=1.1.0
-TPM2_TSS_ENGINE_LINK="https://github.com/tpm2-software/tpm2-tss-engine/archive/v$TPM2_TSS_ENGINE_VER.zip"
+TPM2_OPENSSL_VER="1.1.1"
```

Comment:
why is this 1.1 version openssl here ? Does TPM require 1.1.1 libs to be present in setup ?

Reply:
it's tpm2-openssl lib version